server/pbench/bin/pbench-server-activate-setup-results-host-info

```diff
@@ -46,10 +46,10 @@ if [ ! -z "$testdir" ] ;then
 fi
 cd $documentroot || exit 1
 
-# the directory and filename are specified in the agent config file as the value of
-# [results]/host_info_url 
-# and the setting is duplicated in the server config file: if it changes in one, it
-# *has* to change in the other.
+# the directory and filename are specified in the agent config file as
+# the value of [results]/host_info_url and the setting is duplicated
+# (except for the version number which is appended here)in the server
+# config file: if it changes in one, it *has* to change in the other.
```

Comment:
Do you remember why we need to have this value in two places?

Reply:
One is in the agent and one is in the server config file. The server activation creates the file but the agent has to know where it is.